clients/cobol-lsp-vscode-extension/src/test/suite/testHelper.ts

```diff
@@ -79,6 +79,13 @@ export async function showDocument(workspace_file: string) {
   return editor;
 }
 
+export async function openUntitledDocument(languageId = LANGUAGE_ID) {
+  await vscode.commands.executeCommand(
+    "workbench.action.files.newUntitledFile",
+    { languageId },
+  );
```

Comment:
I think it would be better to `openTextDocument` and `showTextDocument` and return the editor directly.

Reply:
It's good idea to return the editor directly, but why is `openTextDocument` and `showTextDocument` better than `newUntitledFile`?

Reply:
Because showTextDocument returns the editor.

Reply:
Done 👍